### PR TITLE
Add lang to src of media player stories

### PR DESCRIPTION
**Pull request by pjlee11**
Part of https://github.com/bbc/simorgh/issues/3342

**Overall change:** Updates the `src` values for the Media Player storybook examples`

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Automated jest tests added (for new features) or updated (for existing features)~
- ~[ ] This PR requires manual testing~

### CODE REVIEW

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 1.1.1-alpha.7 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Add lang to the embed src URL in storybook examples                       |
```
Comment (pjlee11) [inline packages/components/psammead-media-player/CHANGELOG.md]:
```suggestion
| 1.1.1-alpha.7 | [PR#2312](https://github.com/bbc/psammead/pull/2312) Add lang to the embed src URL in storybook examples                       |
```

File: packages/components/psammead-media-player/src/index.stories.jsx [hunk 2]
```diff
@@ -15,14 +15,14 @@ storiesOf('Components|Media Player', module)
...
-      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp/amp"
+      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp/en/amp"
       placeholderSrc="https://ichef.bbci.co.uk/news/640/cpsdevpb/4eb7/test/ba7482d0-cca8-11e8-b0bf-f33155223fc4.jpg"
     />
   ));
 
 storiesOf('Components|Media Player', module).add('Audio Skin', () => (
   <CanonicalMediaPlayer
-    src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio"
+    src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/ko"
```
Comment (pjlee11) [inline packages/components/psammead-media-player/src/index.stories.jsx]:
```suggestion
    src="https://www.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/ko"
```

File: packages/components/psammead-media-player/src/index.stories.jsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ storiesOf('Components|Media Player', module)
   .add('Audio Skin AMP', () => (
     <AmpMediaPlayer
       isAmp
-      src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/amp"
+      src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/ko/amp"
```
Comment (pjlee11) [inline packages/components/psammead-media-player/src/index.stories.jsx]:
```suggestion
      src="https://www.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/ko/amp"
```

**Review by ryanmccombe — APPROVED** [review]

**Review by ibMadbouly — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]
All looks good in Storybook this end. 👍

I'm curious - what's the reason for removing `test.` from the audio assets?

**Comment by pjlee11** [comment]
@simonsinclair not sure what you mean. 

- [TEST .com](https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio)
- [TEST .co.uk](https://www.test.bbc.co.uk/ws/av-embeds/media/bbc_korean_radio/liveradio)
- [LIVE .com](https://www.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio)
- [LIVE .co.uk](https://www.bbc.co.uk/ws/av-embeds/media/bbc_korean_radio/liveradio)

^ then you can add the lang (EG: `/ko` or `/en-GB`) and `/amp` as a suffix.

Edit: I've removed `test.` because ideally we want to use live data sources when they are available.

**Review by FK78 — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**